falcon/response.py

```diff
@@ -554,6 +554,22 @@ def add_link(self, target, rel, title=None, title_star=None,
         """,
         lambda v: ', '.join(v))
 
+    accept_ranges = header_property(
+        'Accept-Ranges',
+        """
+        Sets the Accept-Ranges headers.
+
+        The "Accept-Ranges" header field allows a server to indicate that it
+        supports range requests for the target resource.
+
+        If the server does not support any kind of range request for the
+        target resource, then set the header to ``none``
```

Comment:
Nitpick: I think it would read better to simple quote "none" rather than formatting it as an identifier.